src/components/data_search/DatasetSearchTableConstants.tsx

```diff
@@ -329,9 +350,12 @@ export const makeDatasetTableHeader = (datasets: DatasetTerm[], selected: number
       cellStyle: makeHeaderStyle(cellWidths.accessType),
       cellDataFn: (dataset: DatasetTerm) => ({
         data: dataset.accessManagement === 'external' ?
-          'External to DUOS' :
+          tooltipIconDisplay(externalAccessIcon, 'external', 'External access request required') :
```

Comment:
This looks like it's duplicating the logic that the filter list uses to map the access management name to the icon path. If the keys are the same in both cases, you could have a single object that maps a name to the icon, type and tooltip text. This code would use all the fields, and the list code would only use the icon path.